core/src/apps/bitcoin/keychain.py

```diff
@@ -15,7 +15,7 @@
     from .authorization import CoinJoinAuthorization
 
     class MsgWithCoinName(Protocol):
-        coin_name = ...  # type: str
+        coin_name: str = ...
```

Comment:
`= ...` is superfluous

Reply:
Addressed in 998eddfeba30f43c6c550f01b717c0c654650ac2